NEWS

```diff
@@ -7,7 +7,46 @@
 ##
 #
 
-## 1.3.0 (February 15, 2018)
+## Current
+
+## 1.4.0-rc1(September 7, 2018)
+
+Features:
+- Improved support for installation with latest ROCm
+- Improved support for latest rdma-core
+- Adding support for CUDA IPC for inter node GPU
+- Added support for latest Mellanox devices
```

Comment:
- Support for multiple connections between same pair of endpoints
- Support large worker address for client/server connection establishment and INARRD_ANY
- Bitwise atomics support
@bureddy anything more for cuda?

Reply:
@yosefe same pair of workers, not endpoint

Reply:
right

Reply:
see if this can be added
-  cuda memory allocation cache for mem type detection